extensions/imessage-2fa/package.json

```diff
@@ -7,19 +7,27 @@
   "author": "yuercl",
   "contributors": [
     "thomas",
-    "arronhunt"
+    "arronhunt",
+    "tzfeng"
   ],
   "categories": [
     "Productivity"
   ],
   "license": "MIT",
   "commands": [
     {
-      "name": "index",
+      "name": "view-2fa-codes",
```

Comment:
Please don't change the name - this will remove the ranking and alias/hotkeys if possible